### ISSUE TITLE
Players lossing items due to NBT tag BlockEntityTag failed checks

### ISSUE THREAD

**Issue by angushushu** [ds58/Panilla#133]
Our config: https://pastebin.com/TqrRBt7k
Players' items disappear with notification although the no-block-entity-tag is false and lenient mode was used.
```
Packet of type PacketPlayOutWindowsItems to player <playername> was dropped because NBT tag BlockEntityTag failed checks
```
The missing items are just some regular items. Some of them are renamed. I've attached a .dat file, which contains two shulkerbox of regular items that will be removed by Panilla.
[10e43b95-10b1-3480-83db-9b0c0983cf99.zip](https://github.com/ds58/Panilla/files/12329728/10e43b95-10b1-3480-83db-9b0c0983cf99.zip)

**Comment by angushushu** [comment]
Though another admin of our server got a different result when she test locally, is that possible to have an option only preventing bookban related stuff but not checking any other item? Players are losing items.

**Comment by ds58** [comment]
Hey, sorry I didn't get to this until now, and thanks for the reminder. This should be fixed in `1.10.2`. I have made a release on SpigotMC so you can download it now.

Also, thanks for the .dat file. That makes things very easy for me!
